src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
So.. I'm not sure why the amd outburst above, but AFAIK encoding support for them should land in ffmpeg in a couple of months. 

FTR back in those days I clearly remember nvidia was the one making hotplates (not to mention especially the 4770 was widely praised for its *freshness* for being the first 40nm card out there)